packages/web3-eth-iban/src/index.ts

```diff
@@ -15,218 +15,9 @@ You should have received a copy of the GNU Lesser General Public License
 along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-import { toChecksumAddress, isAddress, leftPad, hexToNumber, HexString } from 'web3-utils';
-import { InvalidAddressError } from 'web3-errors';
-import { IbanOptions } from './types';
+import { Web3Iban } from './web3_iban';
```

Comment:
```suggestion
import { Iban } from './web3_iban';
```
Make it compatible to 1.x